### DISCUSSION TITLE
Feature for importing raw candle data from exchange

### DISCUSSION THREAD

**Discussion by maxschulmeister** [SockTrader/SockTrader#78] [Ideas]
Make a small feature, where the user can select Exchange, Pair and Daterange. 
Candle data is then downloaded from the exchange and and automatically imported.

**Comment by cwouter** [comment]
Yes this is definitely on the roadmap. For now we'll consider data gathering and data quality as an end-user responsibility. So SockTrader assumes that a stream of candles has no gaps, candle high > candle low etc..

For now we'll consider this a lower prio since a lot of open source tools are already available to solve this problem. That doesn't mean we're not going to include it in a later stadium. But we should first focus on the more important features, which are partially required to actually start working on an import functionality.

Example:
- Create web API
- Create new frontend
- Add database support
